docs/book/v3/coding-style-guide.md

```diff
@@ -1314,10 +1314,10 @@ public function setTitle(string $title, array $context): void
 > `@subpackage` and `@version` MUST NOT be used in comments. Git commits
 > provide accurate information.
 >
-> The words _private_, _protected_, _static_, _constructor_, _deconstructor_,
-> _Created by_, _getter_ and _setter_, MUST NOT be used in comments.
+> The words *private*, *protected*, *static*, *constructor*, *deconstructor*,
+> *Created by*, *getter* and *setter*, MUST NOT be used in comments.
 >
-> The `@var` tag MAY be used in inline comments to document the _Type_
+> The `@var` tag MAY be used in inline comments to document the *Type*
```

Comment:
As this is a major, can we forbid var tag and require inline assertion?

Reply:
I strongly disagree with this

Reply:
Var tags lie. At least assertions fail during tests 🤷‍♂️

Reply:
@Xerkus 
> I strongly disagree with this

These are _inline_ comments, and the idea of inline assertions is a good. Can you explain why you disagree so strongly here?